packages/experiments/src/components/Microfeedback/Microfeedback.base.tsx

```diff
@@ -0,0 +1,145 @@
+  private dislikeRef = React.createRef<HTMLDivElement>();
+  private likeRef = React.createRef<HTMLDivElement>();
+
+  constructor(props: IMicrofeedbackProps) {
+    super(props);
+
+    initializeIcons();
+
+    this.state = {
+      // initial state of icons is neutral and followup is not visible
+      vote: 'no_vote',
+      isFollowupVisible: false
+    };
+  }
+
+  public render() {
+    const likeIcon = this.state.vote === 'like' ? 'LikeSolid' : 'Like';
+    const dislikeIcon = this.state.vote === 'dislike' ? 'DislikeSolid' : 'Dislike';
+    const hideThumbsDownCallout = this.state.vote !== 'dislike' || !this.state.isFollowupVisible;
+    const hideThumbsUpCallout = this.state.vote !== 'like' || !this.state.isFollowupVisible;
+    const onCalloutDismiss = this._onCalloutDismiss.bind(this);
+
+    const classNames = getClassNames(this.props.styles, {
+      theme: this.props.theme
+    });
+
+    return (
+      <div className={classNames.root}>
+        <Stack horizontal styles={microfeedbackStyles}>
+          <div ref={this.likeRef}>
+            <IconButton menuIconProps={{ iconName: likeIcon }} title={this.props.thumbsUpTitle} onClick={this._vote.bind(this, 'like')} />
+          </div>
+          <div ref={this.dislikeRef}>
+            <IconButton
+              menuIconProps={{ iconName: dislikeIcon }}
+              title={this.props.thumbsDownTitle}
+              onClick={this._vote.bind(this, 'dislike')}
+            />
+          </div>
+        </Stack>
+        {this.props.ThumbsUpQuestion ? (
+          <Callout
+            hidden={hideThumbsUpCallout}
+            role="alertdialog"
+            gapSpace={0}
+            target={this.likeRef.current}
+            setInitialFocus={true}
+            onDismiss={onCalloutDismiss}
+          >
+            <div>
+              <FocusZone direction={FocusZoneDirection.vertical}>
+                <Stack padding={10}>
+                  <Text variant="small">{this.props.ThumbsUpQuestion.question}</Text>
+                </Stack>
+                <List items={this.props.ThumbsUpQuestion.options} onRenderCell={this._onRenderCalloutItem} />
+              </FocusZone>
+            </div>
+          </Callout>
+        ) : null}
+        {this.props.ThumbsDownQuestion ? (
+          <Callout
+            hidden={hideThumbsDownCallout}
+            role="alertdialog"
+            gapSpace={0}
+            target={this.dislikeRef.current}
+            setInitialFocus={true}
+            onDismiss={onCalloutDismiss}
+          >
+            <FocusZone direction={FocusZoneDirection.vertical}>
+              <Stack padding={10}>
+                <Text variant="small">{this.props.ThumbsDownQuestion.question}</Text>
+              </Stack>
+              <List items={this.props.ThumbsDownQuestion.options} onRenderCell={this._onRenderCalloutItem} />
+            </FocusZone>
+          </Callout>
+        ) : null}
+      </div>
+    );
+  }
+
+  private _onCalloutDismiss(): void {
```

Comment:
We have a standard to avoid the need for binding. If you define this as a class instanced variable:

```tsx
private _onCalloutDismiss = () => {
```

Then you don't need to do any binding.

Reply:
Fixed